package/src/lib/columns.ts

```diff
@@ -0,0 +1,29 @@
+import format from 'pg-format';
+// Constants
+import { VECTOR_COLUMN } from '../constants.js';
+// Types
+import type { ClientOptions } from '../client.js';
+import type { Hit } from '../index.types.js';
+
+export const getColumns = (clientOptions?: ClientOptions) => {
+  if (!clientOptions?.returnColumns) {
+    return {
+      db: { formatted: '*' },
+      updateHit: (hit: Hit) => {
+        // Remove the vector column here because there is
+        // no easy way to do SELECT * and exclude a column.
+        const { [VECTOR_COLUMN]: _, ...rest } = hit;
+        return rest;
+      },
+    };
+  }
+
+  return {
+    db: {
+      formatted: clientOptions.returnColumns
+        ?.map((col) => format('%I', col))
+        .join(', '),
```

Comment:
Oh that's cool, it was actually a mistake that highlighted returned an array. So ` ${someArrayOfstrings} ` results in comma seperated strings. Really good to know :)
If it's ok can we hold off sharing the return types, because at the moment each lib has no inter-dependency. And, with what I have done in https://github.com/dekimir/postgres-searchbox/pull/12 a common return type wouldn't really work either.

Reply:
Ok.